packages/gasket-nextjs/test/inject-gasket-data.test.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from '@jest/globals';
+import { createElement, Children } from 'react';
+import { injectGasketData } from '../lib/inject-gasket-data.js';
+
+describe('injectGasketData', function () {
+  let mockGasketData, mockHtml;
+
+  beforeEach(function () {
+
+    const Html = (props) => createElement('html', { ...props });
+    const Main = (props) => createElement('main', { ...props });
+    const Head = (props) => createElement('head', { ...props });
+    const NextScript = (props) => createElement('script', { 'data-testid': 'next-script', ...props });
+
+    /**
+     *
+     */
```

Comment:
Either give it a jsdoc or disable since it's a test helper
```suggestion
// eslint-disable-next-line jsdoc/require-jsdoc
```